src/main/system/EvalJson.ts

```diff
@@ -1,7 +1,7 @@
 import { ModuleSpecSchema } from '../schema/ModuleSpec.js';
 
 export const EvalJson = ModuleSpecSchema.create({
-    moduleName: 'Eval.Json',
+    moduleName: 'Eval / Json',
```

Comment:
Change for consistency? Namespaces still recognises `.` right?

Reply:
Yep, all module names are still standardized, so it has no external impact. `/` is obviously canon now